homeassistant/components/linear_garage_door/cover.py

```diff
@@ -0,0 +1,160 @@
+"""Cover entity for Linear Garage Doors."""
+
+from datetime import timedelta
+from typing import Any
+
+from linear_garage_door import Linear
+
+from homeassistant.components.cover import (
+    CoverDeviceClass,
+    CoverEntity,
+    CoverEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import LinearUpdateCoordinator
+
+SUPPORTED_SUBDEVICES = ["GDO"]
+PARALLEL_UPDATES = 1
+SCAN_INTERVAL = timedelta(seconds=10)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Linear Garage Door cover."""
+    coordinator: LinearUpdateCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+    data = coordinator.data
+
+    device_list: list[LinearCoverEntity] = []
+
+    for device_id in data:
+        device_list.extend(
+            LinearCoverEntity(
+                device_id=device_id,
+                device_name=data[device_id]["name"],
+                subdevice=subdev,
+                config_entry=config_entry,
+                coordinator=coordinator,
+            )
+            for subdev in data[device_id]["subdevices"]
+            if subdev in SUPPORTED_SUBDEVICES
+        )
+    async_add_entities(device_list)
+
+
+class LinearCoverEntity(CoordinatorEntity[LinearUpdateCoordinator], CoverEntity):
```

Comment:
New integrations/entities must support [translation of names and attributes](https://developers.home-assistant.io/blog/2023/03/27/entity_name_translations/).

Please adopt your code accordingly.

Reply:
I don't see any names or attributes that need to be translated. Could you please clarify what I need to translate?

Reply:
Sorry, my fault. I got confused with another PR. Nothing to do here